src/main/resources/schemas/schema-form.json

```diff
@@ -115,6 +116,12 @@
           "505": "505 - HTTP_VERSION_NOT_SUPPORTED",
           "507": "507 - INSUFFICIENT_STORAGE"
         }
+      },
+      "gioConfig": {
+        "banner": {
+          "title": "HTTP Status Code send to the consumer in case of validation issues.",
+          "text": "<ul><li>100 - CONTINUE</li><li>101 - SWITCHING_PROTOCOLS</li><li>102 - PROCESSING</li><li>200 - OK</li><li>201 - CREATED</li><li>202 - ACCEPTED</li><li>203 - NON_AUTHORITATIVE_INFORMATION</li><li>204 - NO_CONTENT</li><li>205 - RESET_CONTENT</li><li>206 - PARTIAL_CONTENT</li><li>207 - MULTI_STATUS</li><li>300 - MULTIPLE_CHOICES</li><li>301 - MOVED_PERMANENTLY</li><li>302 - FOUND OR MOVED_TEMPORARILY</li><li>303 - SEE_OTHER</li><li>304 - NOT_MODIFIED</li><li>305 - USE_PROXY</li><li>307 - TEMPORARY_REDIRECT</li><li>400 - BAD_REQUEST</li><li>401 - UNAUTHORIZED</li><li>402 - PAYMENT_REQUIRED</li><li>403 - FORBIDDEN</li><li>404 - NOT_FOUND</li><li>405 - METHOD_NOT_ALLOWED</li><li>406 - NOT_ACCEPTABLE</li><li>407 - PROXY_AUTHENTICATION_REQUIRED</li><li>408 - REQUEST_TIMEOUT</li><li>409 - CONFLICT</li><li>410 - GONE</li><li>411 - LENGTH_REQUIRED</li><li>412 - PRECONDITION_FAILED</li><li>413 - REQUEST_ENTITY_TOO_LARGE</li><li>414 - REQUEST_URI_TOO_LONG</li><li>415 - UNSUPPORTED_MEDIA_TYPE</li><li>416 - REQUESTED_RANGE_NOT_SATISFIABLE</li><li>417 - EXPECTATION_FAILED</li><li>422 - UNPROCESSABLE_ENTITY</li><li>423 - LOCKED</li><li>424 - FAILED_DEPENDENCY</li><li>429 - TOO_MANY_REQUESTS</li><li>500 - INTERNAL_SERVER_ERROR</li><li>501 - NOT_IMPLEMENTED</li><li>502 - BAD_GATEWAY</li><li>503 - SERVICE_UNAVAILABLE</li><li>504 - GATEWAY_TIMEOUT</li><li>505 - HTTP_VERSION_NOT_SUPPORTED</li><li>507 - INSUFFICIENT_STORAG</li></ul>"
```

Comment:
Ha yes indeed. 
If necessary, we can add the same feature as in `x-sh...`. 
The thing that bugs me a bit is that it seems to be supported in the latest version of json-schema
WDYT ?

Reply:
Updated using the new `enumLabelMap` gioConfig